parity/cli/mod.rs

```diff
@@ -883,6 +879,10 @@ usage! {
 			"--warp",
 			"Does nothing; warp sync is enabled by default. Use --no-warp to disable.",
 
+			FLAG flag_fast_and_loose: (bool) = false, or |_| None,
+			"--fast-and-loose",
+			"Does nothing. DB WAL is always activated.",
```

Comment:
Maybe mention here that the option is deprecated?

Reply:
This is just a placeholder really since `--help` does not print these options anymore.